File: problems/j/judgingmoose/judgingmoose.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    l, r = [int(val) for val in input().split()]
    if l == 0 and r == 0:
        print("Not a moose")
    elif l != r:
        print(f"Odd {max([l, r]) * 2}")
    else:
        print(f"Even {l + r}")


if __name__ == '__main__':
    main()

File: problems/j/judgingmoose/judgingmoose.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int l, r, points;

    if (scanf("%d %d", &l, &r) != 2) {
        printf("Failed to read l and r!");
    }

    if (l == 0 && r == 0) {
        printf("Not a moose");
    }
    else if (l != r) {
        points = 0;
        if (l > r) {
            points = l * 2;
        }
        else {
            points = r * 2;
        }
        printf("Odd %d", points);
    }
    else {
        printf("Even %d", l + r);
    }

    return 0;
}